_posts/2016/2016-04-20-jenadevs-docker-party.md

```diff
@@ -0,0 +1,72 @@
+As kittens bustled around the beamer and laptop screens, the Docker newbies built simple web applications using the Python microframework [Flask](http://flask.pocoo.org/) – all running in a container, of course.
+
+### Advanced Workshop
+
+This workshop focused on writing your own Dockerfiles, run the containers and integration testing with [CircleCI](https://circleci.com). Bastian Klein prepared an empty custom [Magento](https://magento.com) `Dockerfile` and `Circle.yml` with comments left only, where the implementation code was completely removed.
+The attendes then had to figure out how the Dockerfile commands needed to be configured under the guidance of the workshop maintainer.
+
+### Special Workshop
+
+The most experienced Dockers gathered together in the special workshop, where the main focus was a bit shifted. We had prepared three worstation laptops. Andreas Grohman introduced CoreOS to the each participant. The goal was to setup a cluster environment with three nodes and run Docker on it.
+Everyone succeeded.
+Afterwards Christian Köhler gave an insight on how to configure Kubernetes with a short demonstration. He also distinctly outlined the advantages to Docker Swarm.
+
+## Summary 
+
+In retrospect the first event of [jenadevs](https://twitter.com/jenadevs) was a total success. Every participant had the possibilty to have fun, meet other devs and learn something new.
+Besides the beginners, which had the possibility to finish their well-documented tutoial at home, all participants could under the cautious eyes of our tutors finish all workshop tasks with happy results. So it was obvious that a lot of guests joined parts of the orga team in the local bar and just talked about the expierience in their developer life, their favorite editors and some nerdier stuff.
+
+Thanks to all attendees and ePages coworkers for making this happen!
+
+## Next Meetup
+
+The goal of the Softwerkskammer Jena is to establish a platform for regular meetings. So we already arranged the next Meetup! This time it is going to be a classical [Coderetreat](http://coderetreat.org) with a [TDD](https://en.wikipedia.org/wiki/Test-driven_development) basics talk and demo upfront.
+Again the participants will be able to choose between different groups where katas in 45-minute-rounds of [coding dojo](http://ccd-school.de/coding-dojo/) are performed. Under certain [constraints](http://coderetreat.org/facilitating/activity-catalog), in [pairs](https://en.wikipedia.org/wiki/Pair_programming) and with different team mates each round.
+
+We are very happy that [Marco Emrich](https://twitter.com/marcoemrich) from the [Softwerkskammer Nürnberg](https://www.softwerkskammer.org/groups/nuernberg) will join us - one of the best trainers and an experienced Coderetreat facilitators - we cannot imagine anyone better to help the participants understand how to the start with the test-first approach!
```

Comment:
...one of the best trainers, and an experienced Coderetreat facilitator. We cannot think of anyone better to help the participants understand how to start with the test-first approach!